.github/workflows/ci.yaml

```diff
@@ -10,6 +10,51 @@ env:
     REQUIRED_PHP_EXTENSIONS: "curl, json"
 
 jobs:
+    coding-standards:
+        name: "Coding Standards"
+
+        runs-on: "ubuntu-latest"
+
+        strategy:
+            matrix:
+                php-version:
+                    - "8.3"
```

Comment:
Thinking about this one. Does it make sense to check on PHP 7.4 only? Same as you did in PHPStan PR.

Reply:
Makes sense
* https://github.com/slunak/pushover-php/pull/48